accounts_frontend/accounts_frontend.model.lkml

```diff
@@ -1,7 +1,3 @@
 connection: "telemetry"
 label: "Firefox Accounts Frontend"
 include: "//looker-hub/accounts_frontend/explores/*"
-# include: "//looker-hub/accounts_frontend/dashboards/*"
-# include: "views/*"
-# include: "explores/*"
-# include: "dashboards/*"
```

Comment:
Just seeing this. Since this is removing commented includes as well, do you think it might be worth changing the default generation logic to something closer to this? 

https://github.com/mozilla/lookml-generator/blob/de2229c087ea44fb86f6d451e80e60cfad06a568/generator/spoke.py#L91-L97

Reply:
I think we might want to remove the default `# include: "views/*"` as this is only needed if an explore is created/extended by a view within the `.model` file itself. In cases where this happens, developers should import only the views they need.

Reply:
> this is only needed if an explore is created/extended by a view within the .model file itself

Per some of my other PR comments, another case where this is needed is when refining a hub view that's used in a hub explore.